quarkc/test/ffi/expected/java/dependencies/pom.xml

```diff
@@ -51,7 +51,7 @@
     <dependency>
       <groupId>quark</groupId>
       <artifactId>quark</artifactId>
-      <version>0.0.1</version>
+      <version>1.0.406</version>
```

Comment:
All these changes to version numbers are not necessary (kinda sorta). I'll explain in person :)